Python/omevv/v1/scripts/autoManageCompliantHosts.py

```diff
@@ -0,0 +1,73 @@
+from typing import Any, Dict, List, Optional, Union
+import argparse
+import sys
+import constants
+import time
+from omevvServerManagement import HostsManagementWrapper
+from omevv_apis_client.models import Credential
+
+import requests
+from requests.packages.urllib3.exceptions import InsecureRequestWarning
+requests.packages.urllib3.disable_warnings(InsecureRequestWarning)
+retry = 3
+
+
+def getHostCompliance(ip, vcusername, vcpassword, vcuuid):
+    global retry
+    headers = {constants.vcGuidHeader: vcuuid}
+    headers["Content-Type"] = "application/json"
+    url = f"https://{ip}/omevv/GatewayService/v1/Consoles/{vcuuid}/Compliance"
+    try:
+        response = requests.get(url, headers=headers, verify=False, auth=(vcusername,vcpassword))
+        response.raise_for_status()
+        return response.json()
+    except Exception as e:
+        print("Exception occured while querying available host compliance ",e," retrying ..")
+        if retry > 0:
+            retry -= 1
+            time.sleep(5)
+            getHostCompliance(ip, vcusername, vcpassword, vcuuid)
+        else:
+            print("Failed after 3 retries,exiting")
+            sys.exit()
+
+
+def getCompliantHosts(discovered_hosts):
+    compliant_hosts = []
+    for i in discovered_hosts:
+        if i["state"] == "COMPLIANT":
```

Comment:
COMPLIANT and NON_COMPLIANT can both be managed. Other states like UNKNOWN, UNSUPORTED, UNDISCOVERED cannot be managed.

NON_COMPLIANT  can also be managed since, the server is non-compliant only on account of being unable to set trap destination.

Reply:
@prasadrao-dell 
The specification/requirements for this issue explicitly to manage only COMPLIANT state systems. 
Please see workflow https://github.com/dell/omivv/issues/32

Reply:
@KristianLamb : It was miss in the workflow, I have updated the workflow now. Kindly update the code.